stan/functions/ct.stan

```diff
@@ -7,21 +9,26 @@ vector ct_threshold_prob(real dt, vector ct_inf_mean, vector ct_inf_sd) {
   }
   return(ldtp);
 }
-vector rel_threshold_prob(vector[] lrit, int t) {
+vector rel_threshold_prob(vector[] lrit, int t, vector ldtp) {
   vector[t] ldtpt;
   for (i in 1:t) {
-    ldtpt[i] = log_sum_exp(lrit[i]);
+    // This log sums lrit (pi) and ldtp (phi) for
+    // the denominator
+    ldtpt[i] = log_sum_exp(lrit[i] + reverse(ldtp));
```

Comment:
I'm not clear this should be reversed? they should both already be in order from infection onwards?